owncloudData/src/main/java/com/owncloud/android/data/spaces/repository/OCSpacesRepository.kt

```diff
@@ -23,16 +24,25 @@ package com.owncloud.android.data.spaces.repository
 
 import com.owncloud.android.data.spaces.datasources.LocalSpacesDataSource
 import com.owncloud.android.data.spaces.datasources.RemoteSpacesDataSource
+import com.owncloud.android.data.user.datasources.LocalUserDataSource
 import com.owncloud.android.domain.spaces.SpacesRepository
 import com.owncloud.android.domain.spaces.model.OCSpace
+import com.owncloud.android.domain.user.model.UserQuota
 
 class OCSpacesRepository(
     private val localSpacesDataSource: LocalSpacesDataSource,
+    private val localUserDataSource: LocalUserDataSource,
     private val remoteSpacesDataSource: RemoteSpacesDataSource,
 ) : SpacesRepository {
     override fun refreshSpacesForAccount(accountName: String) {
         remoteSpacesDataSource.refreshSpacesForAccount(accountName).also { listOfSpaces ->
             localSpacesDataSource.saveSpacesForAccount(listOfSpaces)
+            val personalSpace = listOfSpaces.find { it.isPersonal }
+            personalSpace?.let {
+                val userQuota = UserQuota(accountName, it.quota?.total!!, it.quota?.used!!)
```

Comment:
Is the  `available` property in `UserQuota` (the second parameter here) the equivalent to the `total` property in `SpaceQuota`? I have doubts whether that `available` corresponds to the `total` or to the `remaining`... This deserves a check 👍

Reply:
I've checked how the legacy version was and the `available` value is the `remaining` one, so I changed it ✅  Also the `DrawerActivty` calculates the total with `used` and `available` values.